frontend/src/Components/Admin-form.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+export default class Form extends React.Component {
+    state = {
+        EventsName: '',
+        EventsDate: '',
+        Description: ' '
```

Comment:
It looks like your description defaults to a space inside the string. I would change that to an empty string.